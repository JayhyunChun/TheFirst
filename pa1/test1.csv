240
5
100,100
10,20
500,30
300,40
60,120
