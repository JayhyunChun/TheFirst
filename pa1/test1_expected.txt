1
4,50
0
2
3
